BUILD

```diff
@@ -226,6 +227,7 @@ cc_library(
     "src/core/json/json_common.h",
     "src/core/json/json_reader.h",
     "src/core/json/json_writer.h",
+    "src/core/proto/load_balancer.pb.h",
```

Comment:
Hmm. I'm against scattering proto related stuff all over the tree. How do you feel about src/core/proto/.../xxx.pb.h for the nanopb generated stuff for src/proto/.../xxx.proto?